WordPress/src/main/res/layout/reader_recommended_blogs_card_new.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
+<com.google.android.material.card.MaterialCardView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    style="@style/ReaderCardView"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_margin="@dimen/margin_medium"
+    app:cardBackgroundColor="@color/neutral_0"
```

Comment:
⚠️ It looks like the reader post list recycler uses the same color as this in the background (https://github.com/wordpress-mobile/WordPress-Android/blob/1e900be8988cababe2427abda08d89a28883b462/WordPress/src/main/res/values/reader_styles.xml#L13-L15) so the margin and rounded corners do not show up when displaying this item in the list (see screenshot below). 

![sites-to-follow](https://github.com/wordpress-mobile/WordPress-Android/assets/5091503/b0c9aedf-8da5-43f2-9612-ab5c50309f4c)

I think we need to either set the background of the `ReaderRecyclerView` style to be white (which might be a problem since the spacer between posts in the list will not show up) or wrap this recommended blogs card `MaterialCardView` in a simple `FrameLayout` and set the background of this new root to white.

I would prefer option 2 as it only affects this specific layout.

Reply:
Good catch, thanks! I think I only tested the corner radius on dark mode 😅